psa-crypto-sys/src/extras.rs

```diff
@@ -15,6 +15,22 @@ pub fn PSA_ALG_AEAD_TAG_TRUNCATED_LENGTH(aead_alg: psa_algorithm_t) -> usize {
     (pre_mask_tag_length & TAG_LENGTH_MASK) as usize
 }
 
+/// Retrieves the output size of an ECDH raw key agreement operation shared secret.
+/// Caller must ensure key type is compatible.
+/// This does not match any PSA macro, it will be replaces by PSA_RAW_KEY_AGREEMENT_OUTPUT_SIZE once
+/// mbedTLS adds support for it.
+pub unsafe fn PSA_RAW_ECDH_KEY_AGREEMENT_OUTPUT_SIZE(
+    _key_type: psa_key_type_t,
```

Comment:
Are you keeping it because `PSA_RAW_KEY_AGREEMENT_OUTPUT_SIZE` will use it?

Reply:
I was told by the psa crypto api guys to only bother implementing the ECDH one as Mbed TLS will gain the full macro before it gains support for FFDH. When Mbed TLS adds the macro, this will be removed from `psa-crypto-sys`.

Reply:
oh I meant specifically about the `_key_type` parameter that is not used, sorry.

Reply:
aaah ok. Yes you are right, `PSA_RAW_KEY_AGREEMENT_OUTPUT_SIZE` does require `key_type` so I thought I'd keep the signature the same (even though the name is slightly different).